app/components/DataTable/index.tsx

```diff
@@ -551,6 +555,7 @@ const DataTable = ({
               getRowNodeId={(data) => data.ident}
               onRowDragEnd={canReorder ? onRowDragEnd : null}
               editType="fullRow"
+              enableCellTextSelection
```

Comment:
This is the only line that's relevant to the feature of this branch, everything else is developer debt